### PR TITLE
Add Thai language translation

### PR DESCRIPTION
**Pull request by soap**
I have done translation in Thai as well as add Thai language selection.
Please consider if you want to include or not.

Thanks for your good work.

### CODE REVIEW

**Comment by blomdahldaniel** [comment]
Hi! :)
Glad to see that you want to contribute and exciting with a non latin language! :)

- My first question is why font-awesome is included? Please remove those files if they are not needed for the Thai version?
- Fix indentation in the 2 files i commented.
- Why are not all the text strings translated? Sometimes the backend sections are not translated at all? And in some cases the translations are here and there. Why not be consistent?

File: config/locale.php [hunk 1]
```diff
@@ -36,5 +36,6 @@
         'it'    => ['it', 'it_IT'],
         'pt-BR' => ['pt_BR', 'pt_BR'],
         'sv'    => ['sv', 'sv_SE'],
+		'th'	=> ['th', 'th_TH'],
```
Comment (blomdahldaniel) [inline config/locale.php]: fix indentation :)

File: resources/lang/en/menus.php [hunk 2]
```diff
@@ -79,6 +79,7 @@
             'it' => 'Italian',
             'pt-BR' => 'Brazilian Portuguese',
             'sv' => 'Swedish',
+			'th' => 'Thai',
```
Comment (blomdahldaniel) [inline resources/lang/en/menus.php]: fix indentation :)

File: resources/lang/th/exceptions.php [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+                'confirm' => 'ยืนยันบัญชีผู้ใช้ของคุณ!',
+                'created_confirm' => 'บัญชีของคุณถูกสร้างแล้ว ระบบได้ส่งอีเมลเพื่อให้คุณยืนยันบัญชีของคุณ',
+                'mismatch' => 'รหัสสำหรับยืนยันไม่ตรงกัน',
+                'not_found' => 'รหัสยืนยันไม่มีอยู่ในระบบ',
+                'resend' => 'บัญชีผู้ใช้ของคุณยังไม่ได้รับการยืนยัน กรุณาคลิกที่ลิงค์เพื่อยืนยันในอีเมลที่่ส่งให้ หรือ <a href="' . route('account.confirm.resend', ':user_id') . '">คลิกที่นี่</a> เพื่อให้ระบบส่งลิงค์เพื่อยืนยันบัญชีของคุณอีกครั้งหนึ่ง',
+                'success' => 'บัญชีผู้ใช้ของคุณได้รับการยืนยันแล้ว!',
+                'resent' => 'ระบบได้ส่งอีเมลเพื่อให้คุณได้ทำการยืนยันบัญชีผู้ใช้ของคุณตามชื่ออีเมลที่ระบุไว้แล้ว',
+            ],
+
+            'deactivated' => 'Your account has been deactivated.',
```
Comment (blomdahldaniel) [inline resources/lang/th/exceptions.php]: example of where things have not been translated

File: resources/lang/th/http.php [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+    | The following language lines are used in the views/errors files.
+    |
+    */
+
+    '404' => [
+        'title' => 'ไม่พบหน้าที่ระบุ',
+        'description' => 'เสียใจด้วย แต่หน้าที่คุณเรียกดูไม่มีอยู่',
+    ],
+
+    '503' => [
```
Comment (blomdahldaniel) [inline resources/lang/th/http.php]: lack of translation here as well, wont comment any more places now

**Comment by soap** [comment]
Ok, sorry for that. I will edit them. However, for some text it give better understanding in English than Thai translation. If you want 100% translation, I can do that.

**Comment by rappasoft** [comment]
Yes please. When those are fixed and it's fully translated we will merge!

**Comment by rappasoft** [comment]
Any update @soap?

**Pull request closed without merging**